lib/rspec-puppet.rb

```diff
@@ -14,6 +14,7 @@
 end
 
 RSpec.configure do |c|
+  c.add_setting :environmentpath, :default => '/etc/puppet/environments'
```

Comment:
FYI, puppet 4 is using different file paths. See https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md.